cli/config.go

```diff
@@ -275,6 +276,13 @@ func (h hypervisor) useVSock() bool {
 	return h.UseVSock
 }
 
+func (h hypervisor) guestHookPath() string {
```

Comment:
Please could you add a minimal test in `config_test.go` for this. Looking around your change, it seems we have been a little remiss recently as some of the newer calls are missing unit tests.

You could re-use code from `TestHypervisorDefaultsImage()` - we just need to test the two scenarios - either the default value is returned or the `h.GuestHookPath`.